app/bundles/LeadBundle/Tests/Controller/LeadDetailFunctionalTest.php

```diff
@@ -42,13 +46,48 @@ public function testCustomFieldOrderIsRespected(): void
         $this->em->flush();
         $this->em->clear();
 
+        // initialize lead fields to adjust the expected core labels
+        $lead->setFields([
+            'core' => [
+                'First Name' => [
+                    'value' => 'John',
+                ],
+                'Last Name' => [
+                    'value' => 'Doe',
+                ],
+                'Email' => [
+                    'value' => 'john@his-site.com',
+                ],
+                'Primary company' => [
+                    'value' => null,
+                ],
+                'Points' => [
+                    'value' => 0,
+                ],
+            ],
+        ]);
+        $leadFields = array_filter(
+            $lead->getFields(true),
+            function ($value) {
+                return isset($value['value']);
+            },
+            ARRAY_FILTER_USE_BOTH
+        );
```

Comment:
I'm wondering why `ARRAY_FILTER_USE_BOTH` flag was used as the key is not used in the callback. Also, we can use arrow functions now 🚀 
```suggestion
        $leadFields = array_filter($lead->getFields(true), fn ($value) => isset($value['value']));
```